### PR TITLE
As/ tracking content blocked/allowed status in custom mode

### PR DESCRIPTION
**Pull request by soncuteanca**
# Description

Ensure that ETP Custom mode with option Tracking Content -> In all windows set blocks tracking content
Ensure that ETP Custom mode with option Tracking Content -> Only in Private Windows set allows tracking content

## Bugzilla bug ID

**Testrail: https://testrail.stage.mozaws.net/index.php?/cases/view/446405**
**Link: https://bugzilla.mozilla.org/show_bug.cgi?id=1905200**

## Type of change

Please delete options that are not relevant.

- [x ] New Test

# How does this resolve / make progress on that bug?
Completed

Please describe the progress or significance with respect to the bug listed above.

# Screenshots / Explanations

Please upload any relevant media or add a relevant description with respect to the bug listed above.

# Comments / Concerns

Please add a short blurb about any comments or concerns that this change might cause.

### CODE REVIEW

File: modules/page_object_about_prefs.py [hunk 1]
```diff
@@ -322,3 +322,40 @@ def get_iframe(self) -> WebElement:
...
+        """
+        Configures the Custom mode settings in the Enhanced Tracking Protection (ETP) section by selecting
+        specified options.
+
+        This method first ensures that the browser is set to Custom mode. It then deselects all available
+        tracking protection options (there are checked by default) and selectively re-checks only those specified in the
+        `options_to_check` list (multiple values can be checked at once).
+        """
+        self.set_etp_custom_mode()
+        checkbox_mapping = {
```
Comment (sarinali) [inline modules/page_object_about_prefs.py:341]: I believe the prefs you need are these, though I think I have the wrong values for what you are trying to do, I'm just letting you know if you want to complete with a fixture for prefs u have the choice! :
```
@pytest.fixture()
def add_prefs():
    return [
        ("network.cookie.cookieBehavior", 0),
        ("privacy.trackingprotection.pbmode.enabled", False),
        ("privacy.trackingprotection.cryptomining.enabled", False),
        ("privacy.trackingprotection.fingerprinting.enabled", False),
        ("privacy.fingerprintingProtection.pbmode", False),
    ]
```

**Review by sarinali — DISMISSED** [review]
lgtm, if u want to change the prefs go for it otherwise its fine

**Review by ben-c-at-moz — APPROVED** [review]
Great work!

**Review by sarinali — APPROVED** [review]
LGTM, waiting for checks rn

**Pull request merged**